diagnostic.md

```diff
@@ -6,21 +6,24 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    Router: create routes to each view state
+    Route: define data via models
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    ember g route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    'link-to' allows you to go to the parent directory
+    ie: {{#link-to 'campus'}}Back to Campus{{/link-to}}
+
```

Comment:
Close! `link-to` allows you to link to any defined route in your ember app. Here, we're asking to link to `/campus/boston`, so to get that you'd use `{{#link-to 'campus.boston'}}Boston{{/link-to}}`